### PR TITLE
Aborting with 403 when a user have no orgs

### PR DESCRIPTION
**Pull request by RemiDesgrange**
Maybe we need to make a theme page for 403 :thinking:

### CODE REVIEW

File: ckanext/georchestra/utils/organizations.py [hunk 1]
```diff
@@ -98,6 +99,9 @@ def organization_set_member_or_create(context, user_id, org_id, role):
     :param role:
     :return:
     """
+    # we accept orgs that doesn't exist, but we don't accept user without any orgs.
+    if org_id == '':
+        abort(403)
```
Comment (tonio) [inline ckanext/georchestra/utils/organizations.py]: Looks like passing a message as second argument is possible, wouldn't it be useful?

**Review by tonio — APPROVED** [review]

**Comment by jeanpommier** [comment]
Finally, we don't accept users with no org ? It might be worth looking at the code then, I'm pretty sure we can do some clean up (François asked me to support users with no orgs, I remember writing specific code for this case)

**Comment by RemiDesgrange** [comment]
hum :thinking: @jeanpommier do you want me to have a look ?

**Review by fvanderbiest — CHANGES_REQUESTED** [review]
Users with no Org should be able to browse public CKAN pages without having a 403 ...

File: ckanext/georchestra/utils/organizations.py [hunk 2]
```diff
@@ -98,9 +102,12 @@ def organization_set_member_or_create(context, user_id, org_id, role):
     :param role:
     :return:
     """
+    if org_id == '':
```
Comment (RemiDesgrange) [inline ckanext/georchestra/utils/organizations.py:105]: the interesting part is here :-)

**Comment by fvanderbiest** [comment]
Can you tell us about the consequences of setting `org_id = 'empty_org'` here ?

**Comment by RemiDesgrange** [comment]
It create an org called `empty_org` in ckan. If the `empty_org` exist, it put the user in this org.

File: ckanext/georchestra/utils/organizations.py [hunk 3]
```diff
@@ -1,7 +1,10 @@
 # encoding: utf-8
 
+from __future__ import absolute_import, unicode_literals, print_function
+
 import logging
 import dateutil
+from ckan.lib.base import abort
```
Comment (fvanderbiest) [inline ckanext/georchestra/utils/organizations.py:7]: useless import ?

**Comment by fvanderbiest** [comment]
I don't really like this fix, it's a hacky workaround.
Instead, can we prevent the `organization_set_member_or_create` method from being called when the user has no Org ? cc @jeanpommier

**Pull request closed without merging**